src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -6,7 +6,23 @@
     public static final String OAUTH_AUTHENTICATION_SCHEME = "OAuth";
 
     public static final int VALIDATION_RESULT_MESSAGE_MAX_LENGTH = 2000;
-
+    
+    public static class AccountingStringValidationErrorMessages {
+        public static final String ERROR_CHART_OF_ACCTS_REQUIRED = "Chart is a required field.";
```

Comment:
I agree that the validation service could be an end point in KFS and using the error messages from the properties file would make sense in that case, thanks.